specs/selectors/index.md

```diff
@@ -181,6 +182,16 @@ type ExploreConditional struct {
 	next Selector (rename ">")
 }
 
+## ExploreInterpretAs is a transformation that attempts to 'reify' the current node
+## using an ADL specified by 'as'. ADLs are recognized by agreed-upon strings,
+## similar to libp2p protocols.
+## The ExploreInterpretAs reification process may introduce a data-dependant amount
+## of budget on evaluation based on the specific traversal and ADL implementation.
+type ExploreInterpretAs struct {
+	as String (rename "c")
```

Comment:
I guess I'd consider skipping the rename here considering "as" is already just two bytes?  But don't have a strong opinion of it.

Reply:
we're renaming end to '$'. i'm just gonna run with this and say maybe we got a 'more-compact-than-json' representation of this set of symbols :shrug: